srcpkgs/PolyMC/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'PolyMC'
 pkgname=PolyMC
 version=1.4.1
-revision=1
+revision=2
 build_style=cmake
 configure_args="-DLauncher_BUILD_PLATFORM=Void
  -DLauncher_VERSION_BUILD=${revision}"
-hostmakedepends="extra-cmake-modules openjdk8 pkg-config qt5-host-tools
+hostmakedepends="extra-cmake-modules openjdk17 pkg-config qt5-host-tools
  qt5-qmake scdoc"
 makedepends="qt5-devel"
-depends="virtual?java-runtime qt5-svg qt5-imageformats xrandr"
+depends="openjdk17-jre qt5-svg qt5-imageformats xrandr"
```

Comment:
Why? `virtual?java-runtime` will accept any package that provides `java-runtime` virtual dependency, this includes both `openjdk17-jre` and the [current default of `openjdk8-jre`](https://github.com/void-linux/void-packages/blob/4475c34/etc/defaults.virtual#L25).

If you install `openjdk17-jre` you should be able to remove `openjdk8-jre`, or why not keep both. You could also install `openjdk17-jre` before `PolyMC` and then `openjdk8-jre` won't be pulled in.

Reply:
Ah nvm attempting to install `PolyMC` pulls in `openjdk11-jre` by default for me on `x86_64-musl`, wonder what exactly makes that the case... Either way this is a bit of a nuanced issue, perhaps @classabbyamp can comment better.

I previously suggested [mentioning the existance of PolyMC's README.voidlinux](https://github.com/void-linux/void-packages/pull/34979#issuecomment-1079054056) but it seems that was completely lost in the comments, but the file should already exist as [`/usr/share/doc/steam/README.voidlinux`](https://github.com/void-linux/void-packages/blob/master/srcpkgs/PolyMC/files/README.voidlinux) on your system.

Reply:
> Ah nvm attempting to install `PolyMC` pulls in `openjdk11-jre` by default for me on `x86_64-musl`, wonder what exactly makes that the case... Either way this is a bit of a nuanced issue, perhaps @classabbyamp can comment better.
> 
> I previously suggested [mentioning the existance of PolyMC's README.voidlinux](https://github.com/void-linux/void-packages/pull/34979#issuecomment-1079054056) but it seems that was completely lost in the comments, but the file should already exist as [`/usr/share/doc/PolyMC/README.voidlinux`](https://github.com/void-linux/void-packages/blob/master/srcpkgs/PolyMC/files/README.voidlinux) on your system.

Fair, but README or not, this still means that the package wouldn't have accurate dependencies which puts into question the point of deps in the first place. Not to mention the extra work for new users who'll have to switch one version out for another.